### ISSUE TITLE
alexnet_training_seed_01 appears to provide untrained (random) weights; ImageNet accuracy at chance

### ISSUE THREAD

**Issue by jacob-prince** [brain-score/vision#2459]
I was looking into this model since it's a leading performer on the V1 neural benchmarks.  perhaps I am missing something here... but, surprisingly, it looks like the `.pth` that `alexnet_training_seed_01` loads is indistinguishable from a fresh random init of the arch: ImageNet val top-1 ~0.1% (chance). my analyses suggest that conv1 filters are basically noise, and that the features in the model look untrained as well. The state dict loads cleanly (16/16 keys, right shapes), so I didn't even catch this until I had done some analyses with this model that looked off downstream. it doesn't look like there's any corrupted file, the random weights are what the brainscore API actually provides.  

the trained net exists on an [OSF repo](https://osf.io/3xupm/files/osfstorage): when converting the original TF-Slim `alexnet_v2` checkpoint myself to torch and loading their saved weights, it looks like a normal AlexNet (~54% top-1). so it appears that only `training_seed_01.pth` in the `brainscorevariability` bucket has the issue.  

part of the issue is probably that the model was originally trained in tensorflow.  but this is not a simple transpose/format bug during model loading: layer by layer against the TF checkpoint, per-layer weight correlation is <0.01 under the correct `HWIO -> OIHW` transpose, and the sorted value distributions don't match. they are just unrelated values, i.e. an untrained net, probably saved before the trained weights were loaded? 

I only tested seed_01; the other `alexnet_training_seed_0X` models might be worth checking as well here.

<img width="702" height="484" alt="Image" src="https://github.com/user-attachments/assets/d35fb7dc-86bf-47bd-b497-7dc037610f00" />

one question I have at this time is whether the reported results on brainscore rely on the same code for model loading as in the public-facing API, or whether the model encoding scores / ranking info may depend on other backend code that's not visible to the public?  it would be very interesting if the leading model of V1 neural benchmarks in brainscore was in fact an untrained alexnet (tied with robust-resnet50?) 

<img width="794" height="141" alt="Image" src="https://github.com/user-attachments/assets/6249c4fb-cbd2-4ebe-b1dc-ed90040fd6dc" />

**Comment by jacob-prince** [comment]
here is a code snippet that may help with debugging / loading the saved weights in pytorch and handling the tensorflow conversion:

```
import tensorflow as tf, 
import torch
from model import alexnet_v2_pytorch 

# original TF-Slim alexnet_v2 checkpoint from OSF (Mehrer 2020, osf.io/3xupm)
reader = tf.train.load_checkpoint("model.ckpt_epoch89")
tf2pt = {"conv1": "features.0", "conv2": "features.3", "conv3": "features.6",
         "conv4": "features.8", "conv5": "features.10",
         "fc6": "classifier.0", "fc7": "classifier.3", "fc8": "classifier.6"}

state_dict = {}
for tf_name, pt in tf2pt.items():
    w = torch.from_numpy(reader.get_tensor(f"alexnet_v2/{tf_name}/weights"))   # HWIO
    state_dict[f"{pt}.weight"] = w.permute(3, 2, 0, 1).contiguous()            # -> OIHW
    state_dict[f"{pt}.bias"]   = torch.from_numpy(reader.get_tensor(f"alexnet_v2/{tf_name}/biases"))

model = alexnet_v2_pytorch()
model.load_state_dict(state_dict)   # exact 16/16-key match
model.eval()

# note: this net was trained on [-1,1] inputs: x = (x/255 - 0.5) * 2  (not ImageNet mean/std)
```

**Comment by KartikP** [comment]
Hi @jacob-prince ! Thanks for the thorough report. I was able to reproduce your findings that the `training_seed_01.pth` has the fingerprint of a fresh PyTorch random initialization.

I also checked the other seed checkpoints and they may be similarly affected. Also confirmed that the wrapper uses ImageNet mean/std normalization rather than the original model's [-1, 1] preprocessing.

Regarding your backend question, the leaderboard scoring uses the same public `brainscore_vision.load_model` path. There is no separate training checkpoint used by the backend. The reported V1 scores therefore are the scores for this randomly initialized AlexNet (super fascinating scientific finding!)

The problem, as I see it, is now model identity and provenance. The model identifier and citation imply the trained Mehrer et al., model whereas the supplied weights are just random weights with different preprocessing. There result therefore cannot be interpreted as the score of the trained model.

If you have some time, I would really appreciate the correctly converted trained TF checkpoints to be submitted.

Thanks again for catching this!
